### DISCUSSION TITLE
IP Address Blocked

### DISCUSSION THREAD

**Discussion by jwreplogle** [rustdesk/rustdesk#10386] [General]
I am wondering why my IP address is blocked.  I had my ISP change it and it worked fine for a while.  I have checked firewall and all port open for Rustdesk in/out.
I rarely use Rustdesk but it is nice to have when I need it to help my family members.

**Comment by rustdesk** [comment]
https://www.reddit.com/r/rustdesk/comments/1cr8kfv/should_you_selfhost_a_rustdesk_server/
